apps/web/modules/event-types/views/event-types-single-view.tsx

```diff
@@ -400,38 +400,47 @@ const EventTypePage = (props: EventTypeSetupProps) => {
 
   useEffect(() => {
     const handleRouteChange = (url: string) => {
```

Comment:
This is complex enough to be unit tested now. I think we can refactor out a function from it that accepts `eventType` and url and would give us boolean to inform if navigation should happen or not.